src/main/java/leets/crazyform/domain/workspace/exception/WorkspaceCreationException.java

```diff
@@ -0,0 +1,10 @@
+package leets.crazyform.domain.workspace.exception;
+
+import leets.crazyform.global.error.ErrorCode;
+import leets.crazyform.global.error.exception.ServiceException;
+
+public class WorkspaceCreationException extends ServiceException {
+    public WorkspaceCreationException(String s, Exception e) {
```

Comment:
```suggestion
    public WorkspaceCreationException() {
```

사용 안하는 인자는 삭제해도 될 것 같아요